pkg/monitoring/domainstats/prometheus/prometheus_test.go

```diff
@@ -310,6 +310,84 @@ var _ = Describe("Prometheus", func() {
 			Expect(dto.Gauge.GetValue()).To(BeEquivalentTo(float64(1024)))
 		})
 
+		It("should handle Data_Processed metrics for VMs", func() {
+			ch := make(chan prometheus.Metric, 1)
+			defer close(ch)
+
+			ps := prometheusScraper{ch: ch}
+
+			vmStats := &stats.DomainStats{
+				Cpu:    &stats.DomainStatsCPU{},
+				Memory: &stats.DomainStatsMemory{},
+				MigrateDomainJobInfo: &stats.DomainJobInfo{
+					DataProcessedSet: true,
+					DataProcessed:    1,
+				},
+			}
+			vmi := k6tv1.VirtualMachineInstance{}
+			ps.Report("test", &vmi, vmStats)
+
+			result := <-ch
+			dto := &io_prometheus_client.Metric{}
+			result.Write(dto)
+
+			Expect(result).ToNot(BeNil())
+			Expect(result.Desc().String()).To(ContainSubstring("kubevirt_migrate_vmi_data_processed_bytes"))
+			Expect(dto.Gauge.GetValue()).To(BeEquivalentTo(float64(1024)))
```

Comment:
IMO the `1024` here is a bit confusing, it's hard to understand where it comes from.

I suggest to have something like:
```go
const fakeProcessedData = 1

...

Expect(dto.Gauge.GetValue()).To(BeEquivalentTo(float64(fakeProcessedData*1024)))
```

Reply:
fixed